package.json

```diff
@@ -125,6 +125,10 @@
     "ts-jest": "27.0.7",
     "ts-node": "10.4.0",
     "typescript": "4.3.5"
+  },
+  "engines": {
+    "node": ">=16.0.0",
+    "yarn": ">=1.9.2"
```

Comment:
is this stable with angular 12? according to @LayZeeDK [list](https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3) angular 12 doesn't support node 16 officially?

Reply:
It seems to not be a problem as they updated the engine to be less restrictive, see https://github.com/angular/angular/issues/42076

Reply:
Neither Angular 12.x or Nx 12.x has official support for Node.js 16.

When we read the comment description of https://github.com/iRealNirmal/angular/commit/7f502037ad8bb3bf5e972b92da1ab72e95c3309a, it states that:

>While this change will no longer cause yarn or npm to fail these engine's check, this does
not reflect a change in the officially supported versions of node for Angular.  Angular
continues to maintain support for **Active LTS and Maintenance LTS versions of node**.

Reply:
Sad!

Reply:
@hoebbelsB does something prevent us to migrate to Nx 13 and Angular 13? From what I can recall esm modules break jest setup, isn't it?

Reply:
> From what I can recall esm modules break jest setup, isn't it?

Jest might need some additional setting and I think it's less of a problem in recent major versions of Jest. Not something we can't figure out though.